satellite/metainfo/metainfo.go

```diff
@@ -674,8 +674,13 @@ func (endpoint *Endpoint) GetBucket(ctx context.Context, req *pb.BucketGetReques
 		return nil, status.Errorf(codes.Internal, err.Error())
 	}
 
+	convBucket, err := convertBucketToProto(ctx, bucket)
+	if err != nil {
+		return resp, err
```

Comment:
i think this should return a status error

Reply:
updated